tests/test_debugging.py

```diff
@@ -437,7 +439,11 @@ def circuit(add_bad_snapshot: bool):
         assert result["execution_results"] == expected["execution_results"]
 
         # Make sure shots are overridden correctly
-        result = qml.snapshots(circuit)(add_bad_snapshot=False, shots=200)
+        with pytest.warns(
+            PennyLaneDeprecationWarning,
+            match="'shots' as an argument to the quantum function is deprecated",
+        ):
+            result = qml.snapshots(circuit)(add_bad_snapshot=False, shots=200)
```

Comment:
I don't think this test is intended to test that a deprecation warning is raised, can you update these to use the recommended approach so that the deprecation warning isn't raised?